lib/manageiq/release/repo.rb

```diff
@@ -49,6 +49,7 @@ def fetch(output: true)
 
       def checkout(branch, source = "origin/#{branch}")
         git.reset(:hard => true)
+        git.clean("-xdf")
```

Comment:
🔥 🚽 ❗

Reply:
Hope there wasn't anything important there 😄

Reply:
haha...this is actually the thing causing me problems, while simultaneously fixing other problems. 😩